src/CMakeLists.txt

```diff
@@ -616,6 +624,7 @@ if( MIOPEN_BACKEND MATCHES "OpenCL" OR MIOPEN_BACKEND STREQUAL "HIPOC" OR MIOPEN
         lrn.cpp
         mlo_dir_conv.cpp
         exec_utils.cpp
+        multimarginloss.cpp
```

Comment:
You should sort file name in lexical order.

Reply:
fixed it